mycroft/skills/phillips_hue/__init__.py

```diff
@@ -0,0 +1,283 @@
+                        self.speak_dialog('no.route')
+                        return
+                else:
+                    raise
+
+    def handle_turn_off_intent(self, message):
+        if self.verbose:
+            self.speak_dialog('turn.off')
+        self.all_lights.on = False
+
+    def handle_turn_on_intent(self, message):
+        if self.verbose:
+            self.speak_dialog('turn.on')
+        self.all_lights.on = True
+
+    def handle_activate_scene_intent(self, message):
+        keyword_len = len(message.metadata['ActivateSceneKeyword'])
+        scene_name = message.metadata['utterance'][keyword_len:].strip()
+        if scene_name == '':
+            self.speak_dialog('no.scene.name')
+        else:
+            scene_id = self.bridge.get_scene_id_from_name(
+                scene_name, case_sensitive=False)
+            if scene_id:
+                if self.verbose:
+                    self.speak_dialog('activate.scene',
+                                      {'scene': scene_name})
+                self.bridge.activate_scene(scene_id)
+            else:
+                self.speak_dialog('scene.not.found',
+                                  {'scene': scene_name})
+
+    def handle_decrease_brightness_intent(self, message):
+        if self.verbose:
+            self.speak_dialog('decrease.brightness')
+        brightness = self.all_lights.brightness - self.brightness_step
+        self.all_lights.brightness = brightness if brightness > 0 else 0
+
+    def handle_increase_brightness_intent(self, message):
+        if self.verbose:
+            self.speak_dialog('increase.brightness')
+        brightness = self.all_lights.brightness + self.brightness_step
+        self.all_lights.brightness =\
+            brightness if brightness < 255 else 254
+
+    def handle_decrease_color_temperature_intent(self, message):
+        if self.verbose:
+            self.speak_dialog('decrease.color.temperature')
+        color_temperature =\
+            self.all_lights.colortemp_k - self.color_temperature_step
+        self.all_lights.colortemp_k =\
+            color_temperature if color_temperature > 2000 else 2000
+
+    def handle_increase_color_temperature_intent(self, message):
+        if self.verbose:
+            self.speak_dialog('increase.color.temperature')
+        color_temperature =\
+            self.all_lights.colortemp_k + self.color_temperature_step
+        self.all_lights.colortemp_k =\
+            color_temperature if color_temperature < 6500 else 6500
+
+    def handle_connect_lights_intent(self, message):
+        if not self.config.get('ip', '') == '':
+            self.speak_dialog('ip.in.config')
+            return
+        if self.verbose:
+            self.speak_dialog('connecting')
+        if self._connect_to_bridge():
+            self.speak_dialog('successfully.connected')
+        else:
+            self.speak_dialog('failed.to.connect')
+
```

Comment:
The success message should be spoken here, `_connect_to_bridge` only speaks the success message if it's the first time you're connecting, and need to register with the host. Otherwise it's silent, because it's just being run as part of another request, and there's no need to alert the user that we connected, if they can already see their lights changing.

The failure message here will happen if the ip address and username supplied in the config. The `_connect_to_bridge` method doesn't make any network calls if the ip address is user supplied. It might be better to make a test request against the hub though, and in that method, and handle all the errors/dialog there. I'll look into that.